packages/flutter_tools/lib/src/commands/daemon.dart

```diff
@@ -226,17 +228,27 @@ class DaemonDomain extends Domain {
     registerHandler('shutdown', shutdown);
 
     _subscription = daemon.notifyingLogger.onMessage.listen((LogMessage message) {
-      if (message.stackTrace != null) {
-        sendEvent('daemon.logMessage', <String, dynamic>{
-          'level': message.level,
-          'message': message.message,
-          'stackTrace': message.stackTrace.toString()
-        });
+      if (daemon.logToStdout) {
+        if (message.level == 'status') {
+          print(message.message);
+        } else if (message.level == 'error') {
+          stderr.writeln(message.message);
```

Comment:
This line and the `print(...` statement above seem to be at odds...
one bypasses the zone while the other does not.

Should this line be `printError(...` rather than `stderr.writeln` so that the zone captures the output?
Or should the `print(...` statement above be `stdout.writeln` so that it bypasses the zone?